frame/support/test/tests/pallet_ui/storage_result_query_multiple_type_args.rs

```diff
@@ -0,0 +1,23 @@
+#[frame_support::pallet]
+mod pallet {
+	use frame_support::pallet_prelude::*;
+	use frame_system::pallet_prelude::BlockNumberFor;
+
+	#[pallet::config]
+	pub trait Config: frame_system::Config {}
+
+	#[pallet::pallet]
+	pub struct Pallet<T>(core::marker::PhantomData<T>);
+
+	#[pallet::error]
+	pub enum Error<T> {
+		NonExistentValue,
+		SomeOtherError,
+	}
+
+	#[pallet::storage]
+	type Foo<T: Config> = StorageValue<_, u8, ResultQuery<NonExistentValue, SomeOtherError>>;
```

Comment:
Does this need to support second generic coming from instantiable pallets?

Reply:
It would just look like the following:

```rust
#[pallet::storage]
type Foo<T, I = ()> = StorageValue<_, u8, ResultQuery<NonExistentValue>>;
```

This should just work as-is without any more additional work.